urlshortener.py

```diff
@@ -59,13 +58,14 @@
 )
 
```

Comment:
If you have something to build a better regexp for URLs (forgetting files, localhost, …), feel free to commit it!